_includes/all-versions.html

```diff
@@ -0,0 +1,7 @@
+<section class="row" id="0.0.13">
+    <h2>Version 0.0.13</h2>
+
+    <p>New modules: Equal heights; Vertical divider. Some extra SCSS helpers. Code style improvements and refactoring.</p>
```

Comment:
Semantically it probably should, but getting that to display in a nice way will be slightly annoying. Which is what I meant by this being rough work.